tests/Moq.Tests/VerifyFixture.cs

```diff
@@ -65,6 +66,40 @@ public void ThrowsWithExpressionIfVerifiableExpectationWithLambdaMatcherNotCalle
 			Assert.Contains(@".Execute(It.Is<string>(s => string.IsNullOrEmpty(s)))", mex.Message);
 		}
 
+		[Fact]
+		public void ThrowsWithExpressionIfVerifiableExpectationWithLambdaMatcherVariableNotCalled()
+		{
+			var mock = new Mock<IFoo>();
+
+			Expression<Func<string, bool>> nullOrEmpty = s => string.IsNullOrEmpty(s);
+
+			mock.Setup(x => x.Execute(It.Is(nullOrEmpty)))
+				.Returns("ack")
+				.Verifiable();
+
+			var mex = Assert.Throws<MockException>(() => mock.Verify());
+			Assert.True(mex.IsVerificationError);
+			Assert.Contains(@".Execute(It.Is<string>(s => string.IsNullOrEmpty(s)))", mex.Message);
+		}
+
+		[Fact]
+		public void ThrowsWithExpressionIfVerifiableExpectationWithLambdaMatcherVariableAndClosureAccessNotCalled()
+		{
+			var mock = new Mock<IFoo>();
+
+			string password = "abc123";
+
+			Expression<Func<string, bool>> matchingPassword = s => s == password;
+
+			mock.Setup(x => x.Execute(It.Is(matchingPassword)))
+				.Returns("ack")
+				.Verifiable();
+
+			var mex = Assert.Throws<MockException>(() => mock.Verify());
+			Assert.True(mex.IsVerificationError);
+			Assert.Contains(@".Execute(It.Is<string>(s => s == password)", mex.Message);
```

Comment:
Small detail (but don't worry about it): missing closing parenthesis at the end of the string.